### ISSUE TITLE
WiZ Lights Support

### ISSUE THREAD

**Issue by Z3r10** [hyperion-project/hyperion.ng#1815]
#### What problem does this feature solve?
Adds the support of WiZ lights.
Those lights have pretty good reputation, and are one of the rare that have a local API, making cloud integration virtually optional.
_On the contrary of the many, many Tuya-based bulbs..._
The product lines is also pretty complete, and the company is now backed by Philips (acquired in 2019).

#### What does the proposed API look like?
The local API is not officially documented, but it has been reverse-engineered.
It is possible to find some information on the Internet through many  project.
Examples:
* https://github.com/sbidy/pywizlight
* https://github.com/sbidy/wiz_light

I also built somehow an OpenAPI Spec using Online AI (Perplexity), not sure if it's 100% accurate, but it seems to be a good start.
[WiZ-OpenAPI - v2.yml.txt](https://github.com/user-attachments/files/18162054/WiZ-OpenAPI.-.v2.yml.txt)
_It's possible to easily browse it via Swagger Editor: https://editor.swagger.io/_

#### How should this be implemented in your opinion?
It should be added as a new Hardware LED Instance / Controller Type: Network

#### Are you willing to work on this yourself?
I'm no Professional developer, but I can try.
I would need to learn the Hyperion-ng Code at first.
@Anagorn did some effort a few years ago: https://hyperion-project.org/forum/index.php?thread%2F12411-connect-wiz-light-to-hyperion%2F=

**Comment by Lord-Grey** [comment]
@Z3r10 I am happy to provide you some guidance if you would like to take the endeavour providing Wiz Light support.
Experience tells that without sample devices at hand, it is hard to get an integration up and running (esp. if there is no official documentation). Therefore, I would not spend effort myself  on this one...

You might want reaching out to the Wiz customer/sales team and check, if they would like to share some samples for integration development (and maybe the specs...). For other devices this worked out nicely before....

**Comment by CyberCatStudios** [comment]
My best advice is to get a home assistant box and set up wiz that way. Then, just install the integration and you're good!

**Comment by Z3r10** [comment]
Yep, I though about that, as I already have HA running (with Hyperion integration of course 😁)
2 questions:
* HA does not seem to be supported in the latest Hyperion release, only in the nightly, right?
* I'm concerned about latency: would it be reactive by transiting through HA? My current WS2801 is really fast (I disabled smothing, I like it that way)
